packages/taquito/src/operations/types.ts

```diff
@@ -550,6 +552,22 @@ export interface RPCSmartRollupOriginateOperation {
   parameters_ty: MichelsonV1Expression;
 }
 
+/**
+ * @description RPC failing noop operation
+ */
+export interface RPCFailingNoopOperation {
+  kind: OpKind.FAILING_NOOP;
+  arbitrary: string;
+}
+
+/**
+ * @description Parameters for the `failingNoop` method
+ */
+export interface FailingNoopParams {
+  arbitrary: string;
+  basedOnBlock: BlockIdentifier;
```

Comment:
not really a change request, but I thought we want to point to genesis block instead? 

maybe we could make it an optional param that defaults to 0? what do you think?